model-mesh/samples/servingruntime_custom.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: serving.kserve.io/v1alpha1
+kind: ServingRuntime
+metadata:
+  name: custom-runtime-1.x
+spec:
+  containers:
+    - env:
+        - name: MODEL_DIRECTORY_PATH
+          value: /models
+        - name: MODEL_SERVER_MEM_REQ_BYTES
+          valueFrom:
+            resourceFieldRef:
+              containerName: modelserver
+              resource: requests.memory
+      image: tnarayan74/custom-runtime:6.0
```

Comment:
Is this image right? Or is the thought that since this is an example of a custom runtime we don't actually care what image is here.

Reply:
I think I'll get rid of all the samples here since we don't use them.